src/manifest.json

```diff
@@ -38,6 +38,7 @@
       "id": "extension@getalby.com"
     }
   },
+  "__firefox__content_security_policy": "script-src 'self' blob:; object-src 'self'",
```

Comment:
Maybe put this right below the default entry to make clear that we have both

Reply:
Done!